packages/esm-service-queues-app/src/active-visits/active-visits-table.resource.ts

```diff
@@ -174,82 +126,76 @@ export function usePriority() {
   };
 }
 
+const mapEncounterProperties = (encounter: Encounter): MappedEncounter => ({
+  diagnoses: encounter.diagnoses,
+  encounterDatetime: encounter.encounterDatetime,
+  encounterType: encounter.encounterType.display,
+  obs: encounter.obs,
+  provider: encounter.encounterProviders[0]?.provider?.person?.display,
+  uuid: encounter.uuid,
+  voided: encounter.voided,
+});
+
+export const mapVisitQueueEntryProperties = (
+  queueEntry: QueueEntry,
+  visitQueueNumberAttributeUuid: string,
+): MappedVisitQueueEntry => ({
+  id: queueEntry.uuid,
+  encounters: queueEntry.visit?.encounters?.map(mapEncounterProperties),
+  name: queueEntry.display,
+  patientUuid: queueEntry.patient.uuid,
+  patientAge: queueEntry.patient.person?.age + '',
+  patientSex: queueEntry.patient.person?.gender === 'M' ? 'MALE' : 'FEMALE',
+  patientDob: queueEntry?.patient?.person?.birthdate
+    ? formatDate(parseDate(queueEntry.patient.person.birthdate), { time: false })
+    : '--',
+  // Map `Urgent` to `Priority` because it's easier to distinguish between tags named
+  // `Priority` and `Not Urgent` rather than `Urgent` vs `Not Urgent`
+  priority: queueEntry.priority.display === 'Urgent' ? 'Priority' : queueEntry.priority.display,
```

Comment:
Is this existing code?  If so, that's fine, but seems like we shouldn't be building new functionality based on display values?

Reply:
Nevermind, as I scrolled down I answered my question... we should likely refactor this at some point, as an aside, but nothing to do now